docs/user-guide/out-of-resources/timeout.md

```diff
@@ -255,6 +260,9 @@ To enable this mode, add {ref}`--nondet_difficult_funcs` to the `certoraRun` com
 The minimal difficulty threshold used for the auto-summarization
 can be adjusted using {ref}`--nondet_minimal_difficulty`.
 
+For Solana and Soroban, we recommend summarizing hotspots by enabling munging
+with conditional compilation.
```

Comment:
Do we have an example for this somewhere that we can link to? I believe for external users this might be unclear.